pinephone/vibrator/include/vibrator-impl/FFDevice.h

```diff
@@ -0,0 +1,41 @@
+/* SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (C) 2019 The Android Open-Source Project
+ * Copyright (C) 2021 GloDroid project
+ */
+
+#pragma once
```

Comment:
AOSP seems to be preferring include guards over `#pragma once` (see `hardware/interfaces/vibrator/1.0/default/Vibrator.h`), perhaps it is better to stick to AOSP style, though I personally have no issues with pragma, so this is purely optional.